src/igvfd/tests/data/inserts/prediction_set.json

```diff
@@ -36,6 +36,8 @@
         "award": "1U01HG011952-01",
         "lab": "alan-boyle",
         "status": "released",
-        "uuid": "d84ce280-3014-4c2b-88bb-b7e82352f869"
+        "uuid": "d84ce280-3014-4c2b-88bb-b7e82352f869",
```

Comment:
I can't get this insert to load despite removing the keys to the large_scale_loci_list property in load.xl.

Reply:
what is the error?

Reply:
The insert was failing to load. I've resolved the error for this insert. https://github.com/IGVF-DACC/igvfd/pull/731#discussion_r1458020172